server/server.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const db = require('../database/db');
+// const dataReadFromDisk = require('../database/data');
+
+const app = express();
+const PORT = 4000;
+
+// app.use(bodyParser.json());
+app.use(bodyParser.json({limit: '50mb'}));
+
+app.listen(PORT, function(){
+  console.log(`Listening on port : ${PORT}`);
+})
+
+// ###################Need Refactor
+app.post('/data/ratings_ambience', function(req, res){
+  var queryString = `INSERT INTO ratings_ambience SET ?`;
+  for(var i = 0  ; i < req.body.length ; i++) {
+    db.query(queryString, req.body[i], function (err, results){
+      if (err){
+        console.error('ERROR : Could not seed database');
+        throw err;
+      }
+      console.log('SUCCESS : inserted records into database : ');
+      res.send(results);
+    });
+  }
+});
```

Comment:
you could save things like `req.body` or `req.params.id` to more descriptive variables to make your code more readable